services/worker-manager/src/api.js

```diff
@@ -165,3 +165,32 @@ builder.declare({
   await this.WorkerType.remove({name}, true);
   return res.reply();
 });
+
+builder.declare({
+  method: 'get',
+  route: '/workertypes',
+  query: {
+    continuationToken: /./,
+    limit: /^[0-9]+$/,
+  },
+  name: 'listWorkerTypes',
+  title: 'List All WorkerTypes',
+  stability: APIBuilder.stability.experimental,
+  description: [
+    'TODO',
+  ].join('\n'),
+}, async function(req, res) {
+  const { continuationToken } = req.query;
+  const limit = parseInt(req.query.limit || 100, 10);
+  const scanOptions = {
+    continuation: continuationToken,
+    limit,
+  };
+
+  const data = await this.WorkerType.scan({}, scanOptions);
+
+  if (data.continuation) {
+    data.continuationToken = data.continuation;
```

Comment:
not sure about this 🤔 your thoughts @imbstack ?

Reply:
Fine with me!